docs/usage/basics/projects.md

```diff
@@ -1,8 +1,11 @@
-# Projects
+---
+weight: 2
+description: Project operations, roles and what they allow. Explaining the Four-Eyes-Principle for resource deletion
```

Comment:
It is not really easy to review this PR as changes done in this commit for example are not final. Later on things are restructured again. It would be easier to review if commits contained the final changes.
As you have the changes done locally - does it make sense to split the changes to meaningful commits that can be reviewed separately?